packages/core/src/App/Containers/Routes/routes.jsx

```diff
@@ -44,11 +46,19 @@ class Routes extends React.Component {
 
     render() {
         const { error, has_error, is_logged_in, passthrough } = this.props;
+        const lang = getLanguage();
+        const has_lang = /[?&]lang=/;
```

Comment:
`has_` prefix should be used for booleans. may be we can rename this to `lang_regex`

Reply:
Renamed as suggested